data-platform/autonomous-database/autonomous-json/python-duality-views/files/lab3_crud_operations.py

```diff
@@ -0,0 +1,244 @@
+import oracledb
+
+
+#connect with thick-client
+oracledb.init_oracle_client()
+connection = oracledb.connect(user="myapp",
+                              password="YOUR_PASSWORD",
```

Comment:
Add a note that YOUR_PASSWORD should be changed to a real password!
Better change it to a command line option. Hardcoded values are bad.